presto-main/src/main/java/com/facebook/presto/sql/planner/SqlPlannerContext.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.planner;
+
+public class SqlPlannerContext
+{
+    private int maxNodeInLogicalPlan;
+
+    public SqlPlannerContext(int maxNodeInLogicalPlan)
+    {
+        this.maxNodeInLogicalPlan = maxNodeInLogicalPlan;
+    }
+
+    public int getMaxNodeInLogicalPlan()
+    {
+        return maxNodeInLogicalPlan;
+    }
+
+    public void incNodeNumInLogicalPlan()
```

Comment:
call this incrementPlanNodes

(avoid "inc" and "num" abbreviations)

Reply:
Done